scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/KryoCoder.scala

```diff
@@ -0,0 +1,26 @@
+package com.twitter.scalding.beam_backend
+
+import com.esotericsoftware.kryo.Kryo
+import com.twitter.chill.config.ScalaMapConfig
+import com.twitter.scalding.serialization
+import java.io.{ByteArrayOutputStream, InputStream, OutputStream}
+import org.apache.beam.sdk.coders.AtomicCoder
+
+class KryoCoder[T] extends AtomicCoder[T] {
+  override def encode(value: T, outStream: OutputStream): Unit = {
+    @transient lazy val kryo: Kryo = new serialization.KryoHadoop(new ScalaMapConfig(Map.empty)).newKryo
```

Comment:
I don't think you want to make a kryo for each item, that will be very slow. I think you want a private member variable.

Reply:
+1, we want to keep same kryo instance but somehow made it not to reuse tokens / classIds